pandas/_typing.py

```diff
@@ -158,3 +158,6 @@
 ColspaceArgType = Union[
     str, int, Sequence[Union[str, int]], Mapping[Label, Union[str, int]]
 ]
+
+# random generator
+RandomState = Optional[Union[int, ArrayLike, np.random.BitGenerator, np.random.RandomState]]
```

Comment:
perhaps make `RandomState = Union[...` and then use `random_state: Optional[RandomState] = None,` in generic.py

Reply:
agreed, this shouldn't be Optional (that can be in the signature itself)